src/ui/view/dataverse/component/generalMetadata/generalMetadata.tsx

```diff
@@ -0,0 +1,54 @@
+import type { FC } from 'react'
+import { memo } from 'react'
+import { useTranslation } from 'react-i18next'
+import type { IconName } from '@/ui/component/icon/icon'
+import { Icon } from '@/ui/component/icon/icon'
+import { activeLanguageWithDefault } from '@/ui/languages/languages'
+import { convertToYearIfISODateTime } from '@/util/isoDateTime/isoDateTime'
+import './generalMetadata.scss'
+import './i18n/index'
+
+export type MetadataProperty = {
+  property: string
+  value: string
+  iconName?: IconName
+}
+
+const GeneralMetadataItem: FC<MetadataProperty> = memo(
```

Comment:
Can you explain the purpose of `memo` here. Thanks :pray:

Reply:
In that case it avoids unnecessary re-renders of the `GeneralMetadataItem` by memoizing it's output according to its props, If it's parent re-render for any reason, it won't be re-rendered. However this component is quite small, it might be worse to apply this optimisation mostly on components that are expensive to render.